recipes/harfbuzz/all/conanfile.py

```diff
@@ -41,11 +41,11 @@ class HarfbuzzConan(ConanFile):
 
     def requirements(self):
         if self.options.with_freetype:
-            self.requires("freetype/2.10.2")
+            self.requires("freetype/2.10.4")
         if self.options.with_icu:
             self.requires("icu/67.1")
         if self.options.with_glib:
-            self.requires("glib/2.66.1")
+            self.requires("glib/2.66.2")
```

Comment:
```suggestion
            self.requires("glib/2.67.0")
```

Does it scale to update `harfbuzz` each time `glib` releases a new version (every week?)

Reply:
it does not. I'm doing it only when I notice a conflict of glib requirement. It's true that it can happen quite often, considering the number of packages requiring glib (on CCI: harfbuzz, atk, but many more on on bincrafters). With bincrafters-conventions automatically upgrading bincrafters dependencies, and branches of bincrafters' CCI fork, it is true that it happens quite often currently.
Regarding this bump to glib/2.67.0, it will force to bump atk to 2.67.0, shortly to avoid conflicts in dependents.

Reply:
I'll solve the next conflicts by adding `self.requires("glib/X.Y.Z", override=True)` instead, that should reduce greatly the number of glib bumps in CCI